src/app.py

```diff
@@ -1523,8 +1523,10 @@ def allowable_edit_states(hmuuid):
 def dataset_data_status():
     primary_assays_url = app.config['UBKG_WEBSERVICE_URL'] + 'assaytype?application_context=HUBMAP&primary=true'
     alt_assays_url = app.config['UBKG_WEBSERVICE_URL'] + 'assaytype?application_context=HUBMAP&primary=false'
+    rui_organs_url = app.config['UBKG_WEBSERVICE_URL'] + 'organs?application_context=hubmap'
```

Comment:
@DerekFurstPitt can you use the all uppercase `HUBMAP` instead of `hubmap` in the query string? The ubkg-api uses `HUBMAP` consistently across the code base. The lowercase version may still work, but we don't want to introduce a potential breaking point here.